include/vk_mem_alloc.h

```diff
@@ -6251,6 +6363,7 @@ struct VmaAllocation_T
         void* m_pMappedData; // Not null means memory is mapped.
         VmaAllocation_T* m_Prev;
         VmaAllocation_T* m_Next;
+        mutable VmaWin32Handle m_Handle; // Win32 handle
```

Comment:
Why does it need `mutable`? It looks like a hack. Maybe it would be better to remove some `const`-ness on functions and parameters where needed?

Reply:
Technically it'd be more correct to use mutable on HANDLE, since to the outside user state does not change, but I'll remove the const form function